dependencies.props

```diff
@@ -1,10 +1,39 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
+    <!-- For detailed explanation of the reference versions see:
+
+            https://github.com/dotnet/buildtools/blob/master/Documentation/project-nuget-dependencies.md
+            https://github.com/dotnet/buildtools/blob/master/Documentation/annotated-dependency-props.md
+
+        These ref versions are pulled from https://github.com/dotnet/versions.
+    -->
+    <PropertyGroup>
+        <BuildToolsCurrentRef>bd37cedea957c312fb7d3d681f416d51227d4df1</BuildToolsCurrentRef>
+    </PropertyGroup>
+
     <!-- Package versions used as toolsets --> 
     <PropertyGroup> 
         <FeedTasksPackage>Microsoft.DotNet.Build.Tasks.Feed</FeedTasksPackage> 
         <FeedTasksPackageVersion>1.0.0-prerelease-02121-01</FeedTasksPackageVersion> 
     </PropertyGroup> 
+
+    <!-- Package dependency verification/auto-upgrade configuration. -->
+    <PropertyGroup>
+        <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
+        <DependencyBranch>master</DependencyBranch>
+        <CurrentRefXmlPath>$(MSBuildThisFileFullPath)</CurrentRefXmlPath>
+    </PropertyGroup>
+
     <ItemGroup>
+        <RemoteDependencyBuildInfo Include="BuildTools">
+        <BuildInfoPath>$(BaseDotNetBuildInfo)buildtools/$(DependencyBranch)</BuildInfoPath>
+        <CurrentRef>$(BuildToolsCurrentRef)</CurrentRef>
```

Comment:
Missing indentation--but now I see that this file's apparently 4-spaced rather than the usual 2... any reason for that?

Reply:
That was just the VS Code default for the file.